README.md

```diff
@@ -113,7 +170,7 @@ Description:
     Create react on rails files for install generator.
 ```
 
-We have a repo showing the results of running the generator with various combinations of options, each combination on its own branch: [Generator Results](https://github.com/shakacode/react_on_rails-generator-results-pre-0/pulls).
+We have a repo showing the results of running the generator with various combinations of options, each combination on its own branch: [Generator Results](https://github.com/shakacode/react_on_rails-generator-results-1-0-0/pulls).
```

Comment:
I disagree. We don't have to update this all the time.

Even a little out of date is OK.